### PR TITLE
feat: add eslint-import-resolver-alias package

### PR DESCRIPTION
**Pull request by AndrasEszes**
Now we have alias configuration in the `tsconfig.json`, which can potentially bypass the `import/no-cycle` ESLint rule when using aliases in the codebase. To address this, I have configured the `eslint-import-resolver-alias` package for the project. This change ensures that ESLint can detect import cycles even when we use import aliases.

### CODE REVIEW

File: source/javascripts/core/BitriseYml.ts [hunk 1]
```diff
@@ -12,48 +9,4 @@ type Meta = Required<BitriseYml>['meta'] & {
   };
 };
 
-function deleteWorkflow(yml: BitriseYml, workflowId: string): BitriseYml {
```
Comment (gsanta) [inline source/javascripts/core/BitriseYml.ts:15]: Why is this deleted? Was it unused?

**Review by gsanta — CHANGES_REQUESTED** [review]

File: source/javascripts/core/BitriseYml.ts (hunk 1 above)
Comment (AndrasEszes) [reply source/javascripts/core/BitriseYml.ts:15]: Yes, it is unused. We're using the deleteWorkflow function in models/BitriseYmlService.ts. It hasn't been decided how we'll organize things yet, but everything will be in the right place (hopefully) before we release the new React-only Workflows page. 🙂

Comment (AndrasEszes) [reply source/javascripts/core/BitriseYml.ts:15]: Additionally, this file had an import cycle before the deletion...

**Review by gsanta — APPROVED** [review]
